### DISCUSSION TITLE
how do I disable a view after a callback by it's button

### DISCUSSION THREAD

**Discussion by KlM0RE** [Rapptz/discord.py#7318] [Q&A]
I'm trying to disable a select menu after the user selects an option. but I don't know how to do that.
sorry if my english is kind of weird, i'm still learning.
thanks in advace.

**Comment by Chrovo** [comment]
You can use the `disabled` attribute select menus have, it needs a `bool` value so you can change it after the callback has been called by doing `selectmenu_obj.disabled = True`.
